minecraft/archCommon/build.gradle.kts

```diff
@@ -22,7 +22,7 @@ dependencies {
         isTransitive = false
     }
 
-    modCompileOnly("maven.modrinth:entity-model-features:n0dDeW4R")
+    modCompileOnly("maven.modrinth:entity-model-features:emqHnxMo")
     modCompileOnly("maven.modrinth:entitytexturefeatures:441qELLe")
```

Comment:
Thank you! I'll backport this to 1.21.1
To fix the compilation issue, you'll most likely need to update ETF